.github/workflows/buildah.yml

```diff
@@ -0,0 +1,79 @@
+name: Build dnsmasq oci container image
+
+on:
+  push:
+    branches:
+      - gh-actions
```

Comment:
only to this branch? not main/master?

Reply:
I removed this. It was for testing. I couldn't find another way to make the action run.

Reply:
if you have `workflow_dispatch` then you can start it manually

Reply:
I couldn't, until the job appeared for the first time in the Actions tab. And for that I had to add the push launcher.